.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI/CD
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: 8
```

Comment:
This step is a duplicate of L26-L30.
Do you need both?

Reply:
Yes @hidapple . Both of these steps are required as the workflow relies on both JDK 8 and Maven .